hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/client/rpc/read/TestECBlockReconstructedStripeInputStream.java

```diff
@@ -487,15 +495,8 @@ public void testErrorReadingBlockContinuesReading() throws IOException {
         .randomFill(dataBufs, repConfig.getEcChunkSize(), dataGen, blockLength);
     ByteBuffer[] parity = generateParity(dataBufs, repConfig);
 
-    List<List<Integer>> failLists = new ArrayList<>();
-    failLists.add(indexesToList(0, 1));
-    // These will be the first parity read and then the next parity read as a
-    // replacement
-    failLists.add(indexesToList(2, 3));
-    // First parity and then the data block
-    failLists.add(indexesToList(2, 0));
-
-    for (List<Integer> failList : failLists) {
+    for (int k = 0; k < 5; k++) {
```

Comment:
is this 5 nothing but data+parity ?

Reply:
Not, its just repeating the main loop of the test 5 files to allow a random node to be failed each time and have some randomness in the test.

Reply:
Got it. I have not got full clarity on that test initially....Now I get it. Thanks